source/.vscode/settings.json

```diff
@@ -1,13 +1,12 @@
 // Place your settings in this file to overwrite default and user settings.
 {
-    "files.exclude": {
-        "out": false, // set this to true to hide the "out" folder with the compiled JS files
-        "dist": false // set this to true to hide the "dist" folder with the compiled JS files
-    },
-    "search.exclude": {
-        "out": true, // set this to false to include "out" folder in search results
-        "dist": true // set this to false to include "dist" folder in search results
-    },
-    // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
-    "typescript.tsc.autoDetect": "off"
+  "files.exclude": {
```

Comment:
Let's avoid upload unnecessary files, please add to "files.exclude" the following settings

```
    "build": true,
    "coverage": true,
    "delete_node_modules.bat": true,
    "node_modules": true,
    "package-lock.json": true
```